core/common/src/main/java/alluxio/metrics/MetricKey.java

```diff
@@ -668,7 +668,7 @@ public MetricKey build() {
           .setDescription("Bytes read throughput from Alluxio storage "
               + "via domain socket by this worker")
           .setMetricType(MetricType.METER)
-          .setIsClusterAggregated(true)
+          .setIsClusterAggregated(false)
```

Comment:
why are these all set to false now?

Reply:
Because they are throughput meter. Instead of sending meter one minute rate to the leading master, directly calculating throughput value in the leading master is more correct and easier to understand. The details are written in the PR description